app/views/session/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<% provide :title, "Log in" %>
```

Comment:
```suggestion
<% provide :title, t("log_in") %>
```

Reply:
![image](https://user-images.githubusercontent.com/87749783/131295669-59a71941-6d22-4170-bde9-31cf2966575d.png)
sir if i make the changes then my page name will not show "Log in".

Reply:
Do you use i18n to translate the word "log in "?